tests/models/test_semantic_segmentation.py

```diff
@@ -5,6 +5,7 @@
 # ! The garbage collector
 import gc
 import multiprocessing
+import pathlib
```

Comment:
```suggestion
from pathlib import Path
```

We are using `from pathlib import Path` now. It would be good if we use Path from now on.